circus-train-core/src/main/java/com/hotels/bdp/circustrain/CircusTrain.java

```diff
@@ -90,6 +90,7 @@ public static void main(String[] args) throws Exception {
           .properties("instance.home:${user.home}")
           .properties("instance.name:${source-catalog.name}_${replica-catalog.name}")
           .properties("jasypt.encryptor.password:${password:null}")
+          .properties("housekeeping.schema-name:circus_train")
```

Comment:
This should be done in the housekeeping module but I can't find a cleaner way of setting this property other than this.

Reply:
Understood